tests/wasi-wast/src/wasi_version.rs

```diff
@@ -1,29 +1,39 @@
 pub static ALL_WASI_VERSIONS: &[WasiVersion] = &[WasiVersion::Unstable, WasiVersion::Snapshot1];
 pub static LATEST_WASI_VERSION: &[WasiVersion] = &[WasiVersion::get_latest()];
+pub static NIGHTLY_VERSION: &[WasiVersion] = &[WasiVersion::current_nightly()];
 
 #[derive(Debug, Clone, Copy)]
 pub enum WasiVersion {
     /// A.K.A. Snapshot0
     Unstable,
     Snapshot1,
+    /// This is for making tests pass on Apple M1 while
+    /// still keeping the old test for compatibility reasons
+    CurrentNightly,
```

Comment:
`CurrentNightly` is not deterministic. We need to use a deterministic version for this (that means, an exact one)

Reply:
For example `Nightly_2022_10_18` seems lie a better candidate